tests/test_suites.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from dcqc.enums import TestStatus
+from dcqc.file import FileType
+from dcqc.suites.suite_abc import SuiteABC
+from dcqc.suites.suites import FileSuite, OmeTiffSuite, TiffSuite
+from dcqc.target import Target
+from dcqc.tests.test_abc import TestABC
+from dcqc.tests.tests import LibTiffInfoTest
+
+
+class RedundantFileSuite(TiffSuite):
+    del_tests = (LibTiffInfoTest,)
+
+
+FileType("Unpaired", ())
+
+
+class DummyTest(TestABC):
+    def compute_status(self) -> TestStatus:
+        return TestStatus.NONE
+
+
+def test_that_a_file_suite_results_in_multiple_tests():
+    tests = FileSuite.list_test_classes()
+    assert len(tests) > 0
+    assert all(issubclass(test, TestABC) for test in tests)
+
+
+def test_that_deleting_a_just_added_test_results_in_the_same_test_list():
+    tests_1 = FileSuite.list_test_classes()
+    tests_2 = RedundantFileSuite.list_test_classes()
+    assert set(tests_1) == set(tests_2)
+
+
+def test_that_the_ome_tiff_suite_has_a_superset_of_the_tiff_suite_tests():
+    tiff_tests = TiffSuite.list_test_classes()
+    ome_tiff_tests = OmeTiffSuite.list_test_classes()
+    assert set(ome_tiff_tests) > set(tiff_tests)
+
+
+def test_that_a_test_suite_can_be_retrieved_by_name():
+    actual = SuiteABC.get_suite_by_name("OmeTiffSuite")
+    assert actual is OmeTiffSuite
```

Comment:
We discussed this on the call, but the reason `is` is used because these are not instantiated yet so they are not `isinstance`.